tests/monotouch-test/CoreText/FontManagerTest.cs

```diff
@@ -222,6 +222,9 @@ public void RegisterFontDescriptors_WithCallback ()
 		{
 			TestRuntime.AssertXcodeVersion (11, 0);
 
+			if (TestRuntime.CheckExactXcodeVersion (11, 0, beta: 5))
+				Assert.Ignore ("This began failing for no aparent reason in Beta 5, check back on a later beta.");
```

Comment:
Please issue this and link it to meta issue.

Reply:
Linked here https://github.com/xamarin/xamarin-macios/issues/6690